Bruin Bite.xcodeproj/MenuVC.swift

```diff
@@ -0,0 +1,415 @@
+    //
```

Comment:
this file seems to have been added in by accident, please delete it. NOTE: I am asking you to delete `Bruin Bite.xcodeproj/MenuVC.swift` NOT `Bruin Bite/View Controllers/MenuVC.swift`